### PR TITLE
159915603 ch1 fix issues highlighted during ui code  review session

### PR DESCRIPTION
**Pull request by TheSteelGuy**
**Introduction**
This PR creates incorporates changes to the user interfaces as per feedback

Description of Task to be completed:
Implement challenge

**How to manually test**

- Clone this repository
- navigate into the repository after cloning
- type git checkout 159915603-ch1-fix-issues-highlighted-on-code-UI-review to switch from master to this branch
- navigate further in the UI directory open any index.html file with your preferred browser

Relevant story
[Click here](https://www.pivotaltracker.com/n/projects/2189398/stories/159915603)

### CODE REVIEW

**Comment by kevinsamoei** [comment]
@TheSteelGuy You have merge conflicts. Also, instead of  `Click here` in your PR body, have the ID instead. Your PR title as well has `ch1`. Make it clear what it is.
You were also supposed to fix indentations. Any reason why this has not been implemented?

**Comment by TheSteelGuy** [comment]
I did fix the indentation,, I guess what you see is how the file was before let me send you the code..

File: UI/answer.html [hunk 1]
```diff
@@ -22,65 +21,75 @@ <h2 id="logo">StackOverFLowLite</h2>
 					<a href="../index.html">Logout</a>
 				</li>
 				<li id="right">
-					<a href="profile.html">My profile</a>	
+					<a href="profile.html">My profile</a>
 				</li>
 			</ul>
 		</div>
-		<div class="wrappper">
```
Comment (TheSteelGuy) [inline UI/answer.html:29]: From here no longer exist

File: UI/answer.html [hunk 2]
```diff
@@ -22,65 +21,75 @@ <h2 id="logo">StackOverFLowLite</h2>
...
-											<div id="answerForm">
-												<p id="error"></p>
-												<textarea id="answerBody"> Write Your answer.............. </textarea>
-												<input type="submit" value="Post An Answer">
-											</div>
-										</div>	
-								</div><br><br>
-					</div><!--end row-->
-				</div><!--end body-->
+		<div class="wrappper">
```
Comment (TheSteelGuy) [inline UI/answer.html:28]: Up to here

**Comment by kevinsamoei** [comment]
@TheSteelGuy Some files are still not indented well. Make sure you use 4 spaces.

**Pull request merged**